src/game/shard.rs

```diff
@@ -9,27 +9,15 @@ extern "C" {
     #[wasm_bindgen(js_name = "shard")]
     type Shard;
 
+    /// Current shard name.
     #[wasm_bindgen(js_namespace = ["Game"], js_class = "shard", static_method_of = Shard, getter, js_name = name)]
-    fn name() -> JsString;
+    pub fn name() -> JsString;
```

Comment:
In my opinion this has questionable utility, the most common use I can think of is `game::shard().name() == "shard3"` or similar, which does require the `String` version.